core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -214,7 +216,10 @@ class BpmBillingProjectLifecycle(
   private def cleanupLandingZone(
     landingZoneId: UUID,
     ctx: RawlsRequestContext
-  ): DeleteAzureLandingZoneResult = Try(workspaceManagerDAO.deleteLandingZone(landingZoneId, ctx)) match {
+  ): Option[DeleteAzureLandingZoneResult] = Try(workspaceManagerDAO.deleteLandingZone(landingZoneId, ctx)) match {
+    case Failure(e: ApiException) if e.getCode == HttpStatus.SC_FORBIDDEN =>
```

Comment:
The service returns a 403 instead of a 404 when there's no landing zone present.
It's fine to move on, because if this was a 403 for permission reasons, we won't be able to delete the billing profile anyway, and we don't have a valid instance of a user having access to the billing project but not the underlying resources.

Reply:
It would be good to have this comment in the code.

Reply:
And if we DO happen to get a 404 from the service (which seems more correct), I think we should also handle it and allow the deletion to proceed. Perhaps at some point the 403 will be corrected to a 404.

Reply:
good point